[2024-04-28T15:36:37.514+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-28T15:36:37.633+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_scheduled_dag_20minutes.load scheduled__2024-04-28T15:00:00+00:00 [queued]>
[2024-04-28T15:36:37.688+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_scheduled_dag_20minutes.load scheduled__2024-04-28T15:00:00+00:00 [queued]>
[2024-04-28T15:36:37.690+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2024-04-28T15:36:37.806+0000] {taskinstance.py:2327} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2024-04-28 15:00:00+00:00
[2024-04-28T15:36:37.847+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=79) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-28T15:36:37.878+0000] {standard_task_runner.py:63} INFO - Started process 81 to run task
[2024-04-28T15:36:37.860+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_scheduled_dag_20minutes', 'load', 'scheduled__2024-04-28T15:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpazphy2mq']
[2024-04-28T15:36:37.885+0000] {standard_task_runner.py:91} INFO - Job 64: Subtask load
[2024-04-28T15:36:38.167+0000] {task_command.py:426} INFO - Running <TaskInstance: test_scheduled_dag_20minutes.load scheduled__2024-04-28T15:00:00+00:00 [running]> on host 9ff320b99ca9
[2024-04-28T15:36:38.819+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_scheduled_dag_20minutes' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-28T15:00:00+00:00'
[2024-04-28T15:36:38.824+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-28T15:36:38.829+0000] {logging_mixin.py:188} INFO - Loading data
[2024-04-28T15:36:38.837+0000] {logging_mixin.py:188} INFO - Task failed due to:  name 'display' is not defined
[2024-04-28T15:36:38.838+0000] {logging_mixin.py:188} INFO - {'total_order_value': 0}
[2024-04-28T15:36:38.839+0000] {python.py:237} INFO - Done. Returned value was:                    total_order_value
total_order_value                  0
[2024-04-28T15:36:38.887+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-28T15:36:39.305+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=test_scheduled_dag_20minutes, task_id=load, execution_date=20240428T150000, start_date=20240428T153637, end_date=20240428T153639
[2024-04-28T15:36:39.426+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-28T15:36:39.522+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-28T15:36:39.531+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
